Automattic/wp-calypso
# Plan cancellation copy and style updates

Related to https://github.com/Automattic/dotcom-forge/issues/8751

## Proposed Changes

* This PR updates the plan cancellation copy to clarify the renewal and atomic activation dates.
* It also makes CSS tweaks to vertically align the checkbox, correctly place the checkmark in the box when selected, and avoid word orphans (when `text-wrap: pretty` is supported).

Before | After
--|--
<img width="840" alt="cancel-before" src="https://github.com/user-attachments/assets/282d3344-28e4-4a4a-9ec8-3f3f0fa69db3">  | <img width="797" alt="cancel-after" src="https://github.com/user-attachments/assets/4c0d42be-eb1a-4d9f-95dd-aca4ec6afcfc">


Before | After
--|--
<img width="845" alt="remove-before" src="https://github.com/user-attachments/assets/e70f702c-3bff-4a74-a887-88522e1e0b07">  | <img width="857" alt="remove-after" src="https://github.com/user-attachments/assets/1b8220d6-e7a6-4439-9720-3c43b6bf813d">


## Why are these changes being made?


* To provide clarity around the cancellation flow and fix some design elements.

## Testing Instructions



* Go to /me/purchases/ and select a plan on an Atomic site
* Make sure Auto Renew is on and begin the cancellation process
* When you get to the final cancellation screen, it should look like the screenshot above. (Note: it may have a section about backups depending on your survey selections.)
* Ensure you can cancel plan and there are no regressions.
* Repeat this test, but turn Auto Renew off. There should be no difference between this PR and production.

## Pre-merge Checklist
